apps/trading-e2e/src/integration/market-info.cy.ts

```diff
@@ -129,6 +126,21 @@ describe('market info is displayed', () => {
     validateMarketDataRow(2, 'Scaling Factor', '10');
   });
 
+  it('liquidity displayed', () => {
+    // cy.getByTestId(marketTitle).contains('Liquidity').last().click();
```

Comment:
Chore (non-blocking): Unless there's a compelling reason, might as well remove this commented-out code

Reply:
Removed